src/Dropdown/Dropdown.tsx

```diff
@@ -16,30 +16,32 @@ export interface DropdownItem<T = string> {
     value: T;
 }
 
-export interface DropdownProps {
+export type DropdownProps<T> = {
     id?: string;
     label?: React.ReactNode;
-    items: DropdownItem[];
-    onSelect: (item: DropdownItem) => void;
+    defaultButtonLabel?: string;
+    items: DropdownItem<T>[];
+    onSelect: (item: DropdownItem<T>) => void;
     disabled?: boolean;
-    selectedItem: DropdownItem;
+    selectedItem: DropdownItem<T>;
     numItemsBeforeScroll?: number;
     className?: string;
-}
+};
 
-export default ({
+export default <T,>({
     id,
     label,
+    defaultButtonLabel,
```

Comment:
```suggestion
    defaultButtonLabel = '',
```

By using a default value here, `defaultButtonLabel ?? ''` shouldn't be needed anymore.